### ISSUE TITLE
Add JSON validation

### ISSUE THREAD

**Issue by che85** [QIICR/dcmqi#27]

**Comment by che85** [comment]
@fedorov jsoncpp does not support schema validation which makes it complicated to check for validity. There are some other JSON C++ libraries. 

[rapidjson](https://github.com/miloyip/rapidjson) seems to be pretty handy and supports parsing, generating and valdation of JSON.

What do you think? It's just an idea...

**Comment by che85** [comment]
As discussed with @fedorov adding basic validation to web interface/s for making sure to have at least some validation in case that the user changes the JSON output in place.

**Comment by fedorov** [comment]
I would rephrase this issue as follows:

It would be very helpful to have *any* means of doing at least partial validation of the schema instances against the schema (any = command line or web, JS or python or ruby or whatever).

It would be great to see what are the capabilities of the existing tools that would help us in any way without any extra implementation work. JSON schema site provides a lot of options. It would be nice to understand their existing capabilities.

**Comment by fedorov** [comment]
resolved in #71 and #76
